bluebottle/clients/management/commands/bulk_import.py

```diff
@@ -75,8 +91,7 @@ def handle(self, *args, **options):
                         method_to_call(value)
                     print(" Done!\n")
 
-    def _generic_import(self, instance, data, excludes=False):
-        excludes = excludes or []
+    def _generic_import(self, instance, data, excludes=[]):
```

Comment:
its advisable to not use mutables as default params for functions to avoid unforseen side-effects.
[python gotchas](http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/)